docs/mixtape.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    import pyfixest as pf
</pre><p>Can we add <code>%config InlineBackend.figure_format = "retina"</code> to make the image resolution better :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yep, done 👍